framework/src/Volo.Abp.AspNetCore.Mvc.UI.Packages/Volo/Abp/AspNetCore/Mvc/UI/Packages/Select2/Select2ScriptContributor.cs

```diff
@@ -12,7 +12,6 @@ public override void ConfigureBundle(BundleConfigurationContext context)
         {
             //TODO: Add select2.full.min.js or localize!
             context.Files.AddIfNotContains("/libs/select2/js/select2.min.js");
-            context.Files.AddIfNotContains("/libs/select2/js/select2-bootstrap-modal-patch.js");
```

Comment:
hi @yekalkan 

Don't we need `select2-bootstrap-modal-patch.js` now?

https://github.com/abpframework/abp/pull/3619